doc/themes/nilearn/static/nature.css_t

```diff
@@ -65,6 +65,11 @@ div.content {
 }
 
 
+.alert-danger {
+    background-color: #e74c3c;
+    border-color: #e74c3c;
+    color: #ffffff;
```

Comment:
Although the color is visually striking, the lack of padding makes it feel cramped on the page. I added just a bit and it looks better IMHO :

![Screenshot_2021-02-23 Nilearn Statistical Analysis for NeuroImaging in Python — Machine learning for NeuroImaging(1)](https://user-images.githubusercontent.com/15017191/108935218-5825d100-761b-11eb-80f9-b1d9efa50b03.png)

```suggestion
    color: #ffffff;
    padding: 0.2em;
```